comCam/manual_defects/README.md

```diff
@@ -1,9 +1,13 @@
 This directory contains manual defects for the LSSTComCam instrument.
-The LSSTComCam focal plane consists of one raft of ITL sensors.
-As we do not have QE scans of LSSTComCam, we have simply taken an average of all the ITL sensors from the LSSTCam sensor QE scans as processed in DM-40164.
+
+The defects are a combination of manually identified phosphorescent regions, warm corners, and other column features that are not currently captured by the defect finding code.
 
 Data Processing
 ---------------
 
-Look at ingest_manual_defects.py attached to https://rubinobs.atlassian.net/browse/DM-47365
-for the steps done to process these defects.
+The manual defects were generated by running:
+```
+python $OBS_LSST_DIR/python/lsst/obs/lsst/script/write_comcam_manual_defects.py
+```
+
+The output files then have to be edited by hand to set INSTRUME to "ComCam" instead of "LSSTComCam".
```

Comment:
This should be available as the `instrument.policyName`, I think, but I see that it's capitalized differently.  This works, so I don't think any changes are needed here now.

Reply:
Unfortunately, the `ComCam` for the `INSTRUME` header is neither a policy thing or an instrument thing but a bug in the header service that people decided not to fix.  🤷 
Furthermore, because of the way the calib writing is done, it's impossible to override this (it gets forcefully overwritten at https://github.com/lsst/ip_isr/blob/424a4f6e284da0d1dd0c80969b43ae3a1b70d6f1/python/lsst/ip/isr/calibType.py#L239 )